packages/contentful/src/utils/parse-fields.ts

```diff
@@ -9,13 +9,21 @@ export const addLocaleToFields = (payload: Record<string, unknown>) =>
     {},
   );
 
+export const createLink = (id: string) => ({
+  sys: {
+    type: 'Link',
+    linkType: 'Entry',
+    id,
+  },
+});
+
 export const updateEntryFields = (
   entry: Entry,
   fields: Record<string, unknown>,
 ) => {
-  const updatedEntry = { ...entry };
   Object.entries(fields).forEach(([fieldName, fieldValue]) => {
-    updatedEntry.fields[fieldName] = { 'en-US': fieldValue };
+    // eslint-disable-next-line no-param-reassign
+    entry.fields[fieldName] = { 'en-US': fieldValue };
```

Comment:
It was like this

```
const updatedEntry = { ...entry };
```

before because the first time I've implemented it, I saw the lint error and then I've made this new object `updatedEntry`. But this somehow makes the entry to lose its methods.

I only tested using unit tests when I wrote this. Now, testing manually I saw an error like "entry.publish is not a function" or something like that. So this was actually a bug.